encrypted-media-respec.html

```diff
@@ -159,6 +159,15 @@
 
     <section id="sotd">
         <p>The <code>"persistent-usage-record"</code> session type and the related <em>MediaKeySession destroyed</em> algorithm were removed since the previous version.</p>
```

Comment:
Unrelated: Should this be removed since it no longer applies to the next time the spec is published? (We already published a version (PR) with this text.)